### PR TITLE
Increase max connections

### PR DESCRIPTION
**Pull request by fao89**
Ref: https://github.com/pulp/community/discussions/55
[noissue]

### CODE REVIEW

File: assets/000-postgres.prep [hunk 1]
```diff
@@ -52,6 +53,9 @@ ifthenelse { s6-test -d "${PGDATA}/base" -a -f "${PGDATA}/PG_VERSION" }
   }
 }
 }
+foreground {
+  sed -i "s/max_connections = 100/max_connections = 1024/g" ${PG_CONF_PATH}
```
Comment (mdellweg) [inline assets/000-postgres.prep:57]: Any chance we can do this at image build time?

Comment (fao89) [reply assets/000-postgres.prep:57]: I tried https://github.com/pulp/pulp_ansible/pull/599
but when you try to restart postgres it crashes

Comment (mdellweg) [reply assets/000-postgres.prep:57]: OH, i meant somewhere in the Containerfile but maybe that file is only created on the fly.

**Comment by fao89** [comment]
Closing in favor of https://github.com/pulp/pulp_ansible/pull/599

**Pull request closed without merging**